integration/test_collection.py

```diff
@@ -605,20 +606,82 @@ def test_search_hybrid_only_vector(collection_factory: CollectionFactory) -> Non
     assert len(objs) == 2
 
 
-@pytest.mark.parametrize("limit", [1, 5])
-def test_search_limit(collection_factory: CollectionFactory, limit: int) -> None:
+@pytest.mark.parametrize("limit", [1, 2])
+def test_hybrid_limit(collection_factory: CollectionFactory, limit: int) -> None:
     collection = collection_factory(
-        properties=[Property(name="Name", data_type=DataType.TEXT)],
+        properties=[Property(name="Name", data_type=DataType.TEXT, tokenization=Tokenization.WORD)],
```

Comment:
why tokenozation here?

Reply:
Ah, relic from before I set `alpha=0` and forgot to remove